auth/src/lib.rs

```diff
@@ -289,6 +289,8 @@ pub fn authenticate(profiles: &[profile::Profile], options: Options) -> anyhow::
         }
         sock.to_signer(profile)?
     } else {
+        term::warning("Radicle key won't be added to ssh-agent since it's not running. You can run it with `eval $(ssh-agent)`.");
```

Comment:
I'm not sure I would include instructions on running ssh-agent, as it might be different based on the OS, eg. I think on macOS you'd use launchd.

Reply:
it's also terminal-specific as well. e.g. in fish `eval (ssh-agent -c)`.

do you want it entirely omitted? or should I adjust msg per platform? (would be hard to get 100% right)

Reply:
In order to keep this rather simple, I'd propose to remove the hint entirely and just keep the warning about `ssh-agent` not running.